internal/journald/journald.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+/*
+Package journald provides functions to read and collect journald logs.
+*/
+package journald
+
+import (
+	"context"
+	"os/exec"
+)
+
+type collectionCommand interface {
+	executeCommand() ([]byte, error)
+}
+
+// Command represents a command that is executed by journalctl.
+type Command struct {
+	ctx     context.Context
+	service string
+	command *exec.Cmd
```

Comment:
we shouldn't keep a concrete type in the struct, but an interface that exec.Cmd fulfills, like 
```go
type command interface {
	Output() ([]byte, error)
	String() string
}

```